### PR TITLE
OCLOMRS-558: Fix the issue of missing Add answers field when creating a question concept

### PR DESCRIPTION
**Pull request by alexkayabula**
# JIRA TICKET NAME:
[Fix the issue of missing Add answers field when creating a question concept](https://issues.openmrs.org/browse/OCLOMRS-558)

# Summary:
Fix the issue of missing Add answers field when creating a question concept

### CODE REVIEW

**Comment by dkayiwa** [comment]
Having to update this is more than one place does not smell good.

**Comment by alexkayabula** [comment]
> Having to update this in more than one place does not smell good.

I have changed it to a constant

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import CreateConceptTable from './CreateConceptTable';
 import DescriptionTable from './DescriptionTable';
 import AnswersTable from './AnswersTable';
 import CreateMapping from './CreateMapping';
-import { classes, MAP_TYPE } from './helperFunction';
+import { classes, MAP_TYPE, CLASS_QUESTION } from './helperFunction';
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Is it a class or type?

Comment (alexkayabula) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: It is a class

File: src/tests/dictionaryConcepts/container/CreateConcept.test.jsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ describe('Test suite for dictionary concepts components', () => {
   const props = {
     match: {
       params: {
-        conceptType: 'question',
+        conceptType: CLASS_QUESTION,
```
Comment (dkayiwa) [inline src/tests/dictionaryConcepts/container/CreateConcept.test.jsx]: Is it a class here too?

File: src/tests/dictionaryConcepts/container/EditConcept.test.jsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ describe('Test suite for dictionary concepts components', () => {
   const props = {
     match: {
       params: {
-        conceptType: 'question',
+        conceptType: CLASS_QUESTION,
```
Comment (dkayiwa) [inline src/tests/dictionaryConcepts/container/EditConcept.test.jsx]: Is it a class here too?

File: src/tests/dictionaryConcepts/container/EditConcept.test.jsx [hunk 4]
```diff
@@ -312,7 +312,7 @@ describe('Test suite for mappings on existing concepts', () => {
...
```
Comment (dkayiwa) [inline src/tests/dictionaryConcepts/container/EditConcept.test.jsx]: Is it a class here too?

File: src/tests/dictionaryConcepts/container/CreateConcept.test.jsx (hunk 2 above)
Comment (alexkayabula) [reply src/tests/dictionaryConcepts/container/CreateConcept.test.jsx]: It is a class

File: src/tests/dictionaryConcepts/container/EditConcept.test.jsx (hunk 3 above)
Comment (alexkayabula) [reply src/tests/dictionaryConcepts/container/EditConcept.test.jsx]: It is a class

File: src/tests/dictionaryConcepts/container/EditConcept.test.jsx (hunk 4 above)
Comment (alexkayabula) [reply src/tests/dictionaryConcepts/container/EditConcept.test.jsx]: It is a class

File: src/tests/dictionaryConcepts/container/EditConcept.test.jsx (hunk 3 above)
Comment (dkayiwa) [reply src/tests/dictionaryConcepts/container/EditConcept.test.jsx]: Then why are you assigning it to a concept type yet when it is a class?

Comment (alexkayabula) [reply src/tests/dictionaryConcepts/container/EditConcept.test.jsx]: I have changed it

File: src/tests/dictionaryConcepts/container/CreateConcept.test.jsx [hunk 5]
```diff
@@ -17,7 +17,7 @@ describe('Test suite for dictionary concepts components', () => {
   const props = {
     match: {
       params: {
-        conceptType: 'question',
+        conceptType: 'Question',
```
Comment (dkayiwa) [inline src/tests/dictionaryConcepts/container/CreateConcept.test.jsx]: If this changed from Question to Questions, in how many places will you have to change it?

Comment (alexkayabula) [reply src/tests/dictionaryConcepts/container/CreateConcept.test.jsx]: I have created a constant for it.

File: src/components/dictionaryConcepts/components/helperFunction.js [hunk 6]
```diff
@@ -59,6 +59,14 @@ export const MAP_TYPE = {
   questionAndAnswer: 'Q-AND-A',
 };
 
+export const CONCEPT_TYPE = {
+  question: 'Question',
+};
+
+export const CLASS = {
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/helperFunction.js]: If the above is concept type, shouldn't this be concept class?

Comment (alexkayabula) [reply src/components/dictionaryConcepts/components/helperFunction.js]: I have changed it.

**Pull request merged**